delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutes.scala

```diff
@@ -98,6 +102,24 @@ class BlazegraphViewsRoutes(
                       }
                     )
                   },
+                  // Query a blazegraph view
+                  (pathPrefix("sparql") & pathEndOrSingleSlash) {
+                    implicit val um: FromEntityUnmarshaller[String] =
```

Comment:
why don't you have this at the top? If you don't want to do if for any `String` type, create your own wrapper, like:

```scala
final case class SparqlQuery(value: String) extends AnyVal
object SparqlQuery {
    implicit val um: FromEntityUnmarshaller[String] = ???
}
```

Reply:
in `SparqlClient` change the query signatures to take `SparqlQuery` instead of plain `String`